apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -250,6 +269,7 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
     */
     function vote(uint256 _voteId, bool _supports) external {
         Vote storage vote_ = _getVote(_voteId);
+        // Note: I think you can vote past the last quiet ending period (if no one else has already or if that last person did not again flip the vote)
```

Comment:
Similar to the scenario above, but instead of trying to close the vote, I believe anyone who hasn't voted yet can still vote despite the "phantom" extension having gone by quietly. Note that if this person has enough say to flip the vote again, someone else can do the same later.

Reply:
Addressed in f7a96d8b837a4e03b934a5115638b2a411db2aad